src/bits/bit_vec.rs

```diff
@@ -82,13 +82,22 @@ impl<B: AsRef<[usize]>> Index<usize> for BitVec<B> {
 }
 
 impl BitVec<Vec<usize>> {
-    /// Create a new bit vector of length `len`.
+    /// Create a new bit vector of length `len` initialized to `false`.
```

Comment:
Guidelines: documentation is in third person ("Creates a new bit vector...").

Reply:
Resolved